### DISCUSSION TITLE
[GitHub] 在Traffic中看看有多少人在访问你的仓库

### DISCUSSION THREAD

**Discussion by tyn1998** [X-lab2017/open-wonderland#106] [1️⃣ 新鲜事]
Traffic入口：

<img width="1282" alt="image" src="https://user-images.githubusercontent.com/32434520/175864536-1222db77-5642-470d-92d4-afd05f33c7ce.png">

访问数据：

<img width="945" alt="image" src="https://user-images.githubusercontent.com/32434520/175864821-23dae9d4-3eb6-47f9-8465-91f5d904d79d.png">

**Comment by tyn1998** [comment]
emm，根据[文档](https://docs.github.com/en/repositories/viewing-activity-and-data-for-your-repository/viewing-traffic-to-a-repository)，只有对仓库拥有写权限的人才能看Traffic。

**Comment by xgdyp** [comment]
对应地，api也开放了[traffic](https://docs.github.com/en/rest/metrics/traffic)相关的，这是不是说明我们对于消费者的洞察可以更深一步了

**Reply by will-ww** [reply]
如果是，应该是个不错的数据源~
